PC/winsound.c

```diff
@@ -254,5 +245,10 @@ PyInit_winsound(void)
     ADD_DEFINE(MB_ICONEXCLAMATION);
     ADD_DEFINE(MB_ICONHAND);
     ADD_DEFINE(MB_ICONQUESTION);
+
```

Comment:
```suggestion

#undef ADD_DEFINE

```

Reply:
Not strictly needed, but I prefer to be explicit with macro lifetime.

Reply:
SGTM